docs/manual/common/gettingstarted/IDEs.md

```diff
@@ -1,48 +1,52 @@
+* [Importing Maven projects](#maven)  
+* [Importing sbt projects into Eclipse](#sbt-eclipse)  
+* [Importing sbt projects into IntelliJ](#sbt-intellij)  
 
-If using Maven, use the m2clipse plugin in Eclipse, or the built in IntelliJ Maven support in IntelliJ, to import your project.
 
-## Using sbt
+## <a name="maven"></a>Importing Maven projects
+You can import Maven projects to Eclipse using the m2clipse plugin or use the built in Maven support in IntelliJ.
 
-### Eclipse
+## <a name="sbt-eclipse"></a>Importing sbt projects into Eclipse
 
-[sbt-eclipse](https://github.com/typesafehub/sbteclipse) is needed to generate the Eclipse project files, which are required to import all projects defined in your Lagom build into Eclipse.
+Use the [sbt-eclipse](https://github.com/typesafehub/sbteclipse) plugin to generate the Eclipse project files necessary to import all projects defined in your Lagom build into Eclipse.
 
-If your Lagom build file is in directory `hello`, create a `project/eclipse.sbt` with the following content:
+To add the plugin to an existing project, follow these steps:  
 
-```
-addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "3.0.0")
-```
-
-Save the file. Now, open the terminal, and `cd` to the `hello` directory, and type `sbt`:
-
-```
-$ cd hello
-$ sbt
-... (booting up)
->
-```
+1. Create an `eclipse.sbt` file in your project's `project` directory. For example, if your project is in `my-first-system`, add the `eclipse.sbt` file to `my-first-system/project` on Linux systems, or `my-first-system\project` on Windows systems.  
 
-If you have configured things correctly, typing `eclipse` will generate the Eclipse project files for all projects in your build:
+2. Add the following line to the file and save it:   
 
-```
-> eclipse
-...
-[info] Successfully created Eclipse project files for project(s):
-[info] hello-impl
-[info] hello-api
-[info] hello-stream-impl
-[info] hello-stream-api
-[info] lagom-internal-meta-project-service-locator
-[info] lagom-internal-meta-project-cassandra
->
-```
+  ```addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "3.0.0") ```  
+ 
+3. Open a terminal or command prompt, `cd` to project home, such as the `my-first-system` directory, and start `sbt`:  
+    
+  ```
+     $ cd hello  
+     $ sbt  
+     ... (booting up)  
+     >  
+     ```
+4. Enter `eclipse` to generate the Eclipse project files for all projects in your build:  
+   ```  
+   > eclipse
+   ...  
+   [info] Successfully created Eclipse project files for project(s):  
+   [info] hello-impl  
+   [info] hello-api  
+   [info] hello-stream-impl  
+   [info] hello-stream-api  
+   [info] lagom-internal-meta-project-service-locator  
+   [info] lagom-internal-meta-project-cassandra  
+   
+   >
+   ```
+5. Open Eclipse and follow the [Eclipse instructions](http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.platform.doc.user%2Ftasks%2Ftasks-importproject.htm) for importing existing projects. 
```

Comment:
The code snippets in bullets 2, 3 and 4 are not properly displayed. The code snippet syntax in Markdown has slightly different flavours but the general rule is that there must be (a) a new line, then (b) 3 backticks, then (4) the code, then (d) 3 backticks and finally (e) another newline. 

The current look of the docs when I run it locally is:
<img width="668" alt="screen shot 2017-01-11 at 12 27 04" src="https://cloud.githubusercontent.com/assets/762126/21847160/a87c134e-d7fa-11e6-9a7b-7fc298e4cbda.png">
but I think it could look like:
<img width="682" alt="screen shot 2017-01-11 at 12 33 22" src="https://cloud.githubusercontent.com/assets/762126/21847167/b4afba62-d7fa-11e6-95dc-838b9c23bf27.png">
which is closer to what we already have on http://www.lagomframework.com/documentation/1.2.x/java/IDEs.html.

The problem is that code snippets rendering is a competing feature with numbered lists (radio bullets). If you compare the two screenshots above, on one there's incremental values on the numbered bullets but code is not properly rendered, while on the other the numbered bullets count is restarted but the code is displayed correctly.

The issue here is the markdown processor interpreting every line that start with `3. ` (digit+period+blankspace) as being part of a numbered list and not being able to detect that our `2. ` and `3. ` belong to the same numbered list because there a code snippet in the middle. I think there'a a trade-off and we will have to choose nice code and no numbered lists (unnumbered lists should work fine though).